datafusion/core/src/datasource/listing/table.rs

```diff
@@ -50,7 +50,7 @@ use crate::{
     error::{DataFusionError, Result},
     execution::context::SessionState,
     logical_expr::Expr,
-    physical_plan::{empty::EmptyExec, project_schema, ExecutionPlan, Statistics},
```

Comment:
Long term I would like to cut the dependency between physical_plan and `datasource` -- but I need to get physical_plan extracted first